src/cli/cli.cpp

```diff
@@ -2742,6 +2766,19 @@ template <> otError Interpreter::Process<Cmd("log")>(Arg aArgs[])
         }
     }
 #if (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_DEBUG_UART) && OPENTHREAD_POSIX
```

Comment:
This one doesn't come through the import because OPENTHREAD_POSIX isn't set in our Doxygen config